pytorch_lightning/trainer/training_loop.py

```diff
@@ -288,7 +288,9 @@ def train(self):
         model = self.get_model()
 
         # load data
-        self.reset_train_dataloader(model)
+        # if reload_dataloaders_every_epoch, this is moved to the epoch loop
+        if not self.reload_dataloaders_every_epoch:
+            self.reset_train_dataloader(model)
         self.reset_val_dataloader(model)
```

Comment:
Will the same problem happen for val_dataloader as well?

Reply:
val_dataloader doesn't get re-initialized every epoch with reload_dataloaders_every_epoch, the name is a bit misleading as ethanwarris mentioned (and I'm not sure it should)

Reply:
that was my question above... :]

Reply:
good point @ethanwharris since val dataloader does reset every epoch anyhow.

Reply:
@TevenLeScao pls ^^

Reply:
Yeah it's in the latest commit ! Passing CI atm